packages/react/src/section-alert/docs/overview.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Section alert
+description: Section alerts are non-disruptive notifications that provide information and messages about a state change in a section of a page.
+group: Content
+storybookPath: /story/Content-SectionAlert--basic
+relatedComponents: ['page-alert', 'global-alert']
+---
+
+Section alerts appear inside of the main content area of the page and are used to alert users to a particular section of the screen. They should not be confused with Callouts.
+
+```jsx live
+<SectionAlert title="Your changes have been saved" tone="success" />
+```
+
+<DoHeading />
+
+- use to notify users about a change in state in a section of a
+  page
+- use to communicate error or success messages
+- allow it to be dismissed if it is not critical.
+- place above interactive components or at the top of page sections to show information or status
+- use constructive no-blame language, avoid vague descriptions and always provide a solution
+- keep it short - content should be understood at a glance
+- use the appropriate status of success, info, error or warning
+
+<DontHeading />
+
+- use to communicate messages which relate to the entire page - use [Page alert](/components/page-alert) instead
+- use to communicate messages which relate to the entire application - use [Global alert](/components/global-alert) instead
+- use to contain text in articles or other blocks of text content - use [Callout](/components/callout) instead
+- hide the message automatically. Let the user dismiss the message instead.
+- use for form errors - use [Page alert](/components/page-alert) instead
```

Comment:
I'm not sure about this point @nathsimpson. I think there is a chance that section alert will be used for micro-form validation in multi-action pages